### PR TITLE
Split tasks

### PR DESCRIPTION
**Pull request by tomphp**
# Motivation and Context
This is WIP on the load test pipeline - merging it to try and keep the deltas from getting any bigger.

### CODE REVIEW

File: scripts/loadtest-scripts/execute_collection_exercise.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+SURVEY_ID = os.getenv('SURVEY_ID', '75b19ea0-69a4-4c58-8d7f-4458c8f43f5c')
+SURVEY_CLASSIFIERS = os.getenv('SURVEY_CLASSIFIERS',
+                               '{"form_type":"0102","eq_id":"1"}')
+
+PERIOD_OVERRIDE = os.getenv('COLLECTION_EXERCISE_PERIOD', None)
+
+logger.initialise_from_env()
+
+config = sdcclient.config_from_env()
+sdc = SDCClient(config)
```
Comment (mfcrocker) [inline scripts/loadtest-scripts/execute_collection_exercise.py:21]: It feels a bit weird to get config from sdcclient and then pass it to SDCClient

File: scripts/loadtest-scripts/execute_collection_exercise.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    wait_for(lambda: sdc.collection_exercises.get_state(exercise_id) in [
+        'READY_FOR_REVIEW'])
+
+    sdc.collection_exercises.execute(exercise_id)
+
+    wait_for(lambda: collection_exercise_is_ready_for_live(exercise_id))
+
+    today = datetime.date.today().strftime('%d%m%Y')
+
+    print(json.dumps(
```
Comment (mfcrocker) [inline scripts/loadtest-scripts/execute_collection_exercise.py:125]: Would the next stage pickup still work with logging.info?

**Review by mfcrocker — APPROVED** [review]
Looks good!

**Review by PurpleBooth — APPROVED** [review]

File: scripts/loadtest-scripts/execute_collection_exercise.py (hunk 2 above)
Comment (tomphp) [reply scripts/loadtest-scripts/execute_collection_exercise.py:125]: Yes, all logging is going to `stderr`

File: scripts/loadtest-scripts/execute_collection_exercise.py (hunk 1 above)
Comment (tomphp) [reply scripts/loadtest-scripts/execute_collection_exercise.py:21]: Fair point, this was a hangover from a previous requirement

**Pull request merged**